From 281d9dfff3a98883b67184f80ab50d208cc3d641 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Wed, 8 Aug 2018 14:43:48 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Enable CONFIG_DM_VERITY_AVB for
 avb

This is needed to enable the AVB specific dm-verity error handler.

Change-Id: I331ba1e6f459ec51ff8584cf1471ad05dd90f6db
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index ebc02e10fbb0..63f0e2718b0c 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -258,6 +258,7 @@ CONFIG_DM_CRYPT=y
 CONFIG_DM_UEVENT=y
 CONFIG_DM_VERITY=y
 CONFIG_DM_VERITY_FEC=y
+CONFIG_DM_VERITY_AVB=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
 # CONFIG_NET_VENDOR_3COM is not set
-- 
2.35.3

